### PR TITLE
Don't add roles on init

### PR DESCRIPTION
**Pull request by mjangda**
add_role can lead to front-end writes which are not ideal. It can lead to undesirable circumstances with other roles being removed if there are intermittent database issues.

We still trigger our upgrade routine on `admin_init` so new installs will still get the roles (and there's still a chance for the latter issue to trigger but the likelihood is significantly reduced).

Also includes a convenience WP-CLI command to force run the upgrade routine if needed (`wp vipsupport reset-roles`)

## To Test

On a new install (with PR in place):

1. If you examine the `wp_user_roles` option in the db, in should not include either of the `vip_support_*` roles.
1. Goto wp-admin
1. This should trigger the upgrade routine
1. The `wp_user_roles` option should now have the roles (can also confirm with `wp role list`.

WP-CLI command:

1. Run `wp vipsupport reset-roles`; should result in no changes
1. Run `wp role delete vip_support`. Running `wp role list` should show `vip_support` role missing.
1. Run `wp vipsupport reset-roles` again. `vip_support role should be restored.

### CODE REVIEW

**Comment by mjangda** [comment]
Looks like tests might need some tweaks

File: bin/download-wp-tests.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```
Comment (mjangda) [inline bin/download-wp-tests.sh:1]: Copied over from mu-plugins.

File: bin/install-wp-tests.sh [hunk 2]
```diff
@@ -5,34 +5,20 @@ if [ $# -lt 3 ]; then
 	exit 1
```
Comment (mjangda) [inline bin/install-wp-tests.sh:5]: Copied over from mu-plugins.

File: bin/phpunit-docker.sh [hunk 3]
```diff
@@ -1,29 +1,48 @@
 #!/bin/bash
```
Comment (mjangda) [inline bin/phpunit-docker.sh:1]: Copied over from mu-plugins

File: bin/utils.sh [hunk 4]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
```
Comment (mjangda) [inline bin/utils.sh:1]: Copied over from mu-plugins

**Comment by nickdaugherty** [comment]
The test setup pain makes me think we should just fold this back into mu-plugins directly (at a later time of course).

**Comment by mjangda** [comment]
> The test setup pain makes me think we should just fold this back into mu-plugins directly (at a later time of course).

Yep, that's been a long-standing task unfortunately.

**Review by nickdaugherty — APPROVED** [review]
LGTM

**Pull request merged**